pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -374,83 +376,53 @@ func (f *framework) RunScorePlugins(pc *PluginContext, pod *v1.Pod, nodes []*v1.
 			}
 		}
 	})
-
 	if err := errCh.ReceiveError(); err != nil {
 		msg := fmt.Sprintf("error while running score plugin for pod %q: %v", pod.Name, err)
 		klog.Error(msg)
 		return nil, NewStatus(Error, msg)
 	}
 
-	return pluginToNodeScores, nil
-}
-
-// RunNormalizeScorePlugins runs the NormalizeScore function of Score plugins.
-// It should be called after RunScorePlugins with the PluginToNodeScores result.
-// It then modifies the list with normalized scores. It returns a non-success Status
-// if any of the NormalizeScore functions returns a non-success status.
-func (f *framework) RunNormalizeScorePlugins(pc *PluginContext, pod *v1.Pod, scores PluginToNodeScores) *Status {
-	ctx, cancel := context.WithCancel(context.Background())
-	errCh := schedutil.NewErrorChannel()
+	// Run NormalizeScore method for each ScoreWithNormalizePlugin in parallel.
```

Comment:
why not keep the functions and just make them local (normalizeScores and applyScoreWeights)?

Reply:
I just found the 3 steps are very similar (parallel execution and receive error) and it looks good to me to put them in the same function (saves a few lines of code). But no particular reason really.

If you like the separately, I can change it.